web/client/utils/GeoStoryUtils.js

```diff
@@ -496,6 +497,78 @@ export const parseHashUrlScrollUpdate = (url, hash = '', storyId, sectionId, col
     return null;
 };
 
+// customGetEntityId is a utility function used by html-to-draftjs library in order
+// to create anhor tag chunks
+export const customGetEntityId = (node) => {
+    let entityId = undefined;
+    if (
+        node instanceof HTMLAnchorElement
+    ) {
+        const entityConfig = {};
+        if (node.dataset && node.dataset.mention !== undefined) {
+            entityConfig.url = node.href;
+            entityConfig.text = node.innerHTML;
+            entityConfig.value = node.dataset.value;
+            entityId = Entity.__create(
+                'MENTION',
+                'IMMUTABLE',
+                entityConfig,
+            );
+        } else {
+            entityConfig.url = node.getAttribute ? node.getAttribute('href') || node.href : node.href;
+            entityConfig.title = node.innerHTML;
+            entityConfig.targetOption = node.target;
+            const interactionType = node.getAttribute && node.getAttribute('data-geostory-interaction-type');
+            const interactionParams = node.getAttribute && node.getAttribute('data-geostory-interaction-params');
+
+            const attributes = {
+                type: interactionType,
+                params: interactionParams
+            };
+
+            entityConfig.attributes = attributes;
+            entityId = Entity.__create(
+                'LINK',
+                'MUTABLE',
+                entityConfig,
+            );
+        }
+    }
+    return entityId;
+};
+
+// customEntityTransform is a utility function used by draftjs-to-html inorder to create html from
+// entities in draftjs
+export const customEntityTransform = (entity, text) => {
+    if (entity.type === 'MENTION') {
+        return `< href="${entity.data.url}" class="wysiwyg-mention" data-mention data-value="${entity.data.value}">${text}</a>`;
+    }
+
+    if (entity.type === 'LINK') {
+        const targetOption = entity.data.targetOption || '_self';
+        const attributes = entity.data.attributes;
+
+        if (attributes && attributes.type === "scrollTo") {
+            return `<a data-geostory-interaction-type="${attributes.type}" data-geostory-interaction-params="${attributes.params}" onclick="__geostory_interaction(type='scrollTo', '${attributes.params}')">${text}</a>`;
```

Comment:
If we apply the previously suggested changes to attributes, we need also to update this because now attributes contains

 data-geostory-interaction-type and data-geostory-interaction-params